docs/book/src/topics/managedcluster.md

```diff
@@ -136,6 +136,7 @@ spec:
   mode: System
   osDiskSizeGB: 512
   sku: Standard_D2s_v3
+  maxPods: 24
```

Comment:
```suggestion
```
nit: This section does not have any optional field, would be great to follow the same here wdyt ??

Reply:
done